source/plugins/ruby/kubelet_utils.rb

```diff
@@ -41,6 +41,55 @@ def get_node_capacity
       end
     end
 
+    def get_node_allocatable
+      begin
+        cpu_capacity = 1.0
+        memory_capacity = 1.0
+        cpu_allocatable = 1.0
+        memory_allocatable = 1.0
+
+        capacity_response = CAdvisorMetricsAPIClient.getAllMetricsCAdvisor(winNode: nil)
+        response = CAdvisorMetricsAPIClient.getCongifzCAdvisor(winNode: nil)
+        if !response.nil? && !response.body.nil?
+          kubereserved_cpu = JSON.parse(response.body)["kubeletconfig"]["kubeReserved"]["cpu"]
+          @log.info "get_node_allocatable::kubereserved_cpu  #{kubereserved_cpu}"
+          kubereserved_memory = JSON.parse(response.body)["kubeletconfig"]["kubeReserved"]["memory"]
+          @log.info "get_node_allocatable::kubereserved_memory #{kubereserved_memory}"
+          systemReserved_cpu = JSON.parse(response.body)["kubeletconfig"]["systemReserved"]["cpu"]
+          @log.info "get_node_allocatable::systemReserved_cpu  #{systemReserved_cpu}"
+          systemReserved_memory = JSON.parse(response.body)["kubeletconfig"]["systemReserved"]["memory"]
+          @log.info "get_node_allocatable::systemReserved_memory #{systemReserved_memory}"
+          evictionHard_cpu = JSON.parse(response.body)["kubeletconfig"]["evictionHard"]["nodefs.available"]
+          @log.info "get_node_allocatable::evictionHard_cpu #{evictionHard_cpu}"
```

Comment:
https://docs.microsoft.com/en-us/azure/aks/custom-node-configuration

kubelet config changer: keep track of this.